lcm-cmake/lcmUtilities.cmake

```diff
@@ -260,6 +265,10 @@ function(lcm_wrap_types)
       if(_line MATCHES "^ *package +")
         # Get package name
         _lcm_extract_token(_package 1 "${_line}")
+        # Append package prefix if it exists
+        if(_package_prefix)
```

Comment:
Why not `if(DEFINED _PACKAGE_PREFIX)` and get rid of the unnecessary duplicated variable?

Nit: "Append package prefix *if one was specified*" (emphasis to show changed text)